pages.pt_BR/linux/htop.md

```diff
@@ -0,0 +1,15 @@
+# htop
+
+> Visualizar informação dinâmica em tempo real acerca de processos em execução. Uma versão melhorada do comando `top`.
```

Comment:
Nós tentamos não utilizar de adjetivos durante a descrição dos comandos. 
```suggestion
> Visualizar informação dinâmica em tempo real acerca de processos em execução. Uma alternativa para o comando `top`.
```

Reply:
Que tal? "Uma variante do comando `top` com recursos adicionais"

Reply:
```suggestion
> Visualizar informações em tempo real sobre os processos em execução. Uma variante do comando `top` contendo recursos adicionais.
```

Reply:
This is fine as is, it is consistent with the English version, no need to change anything.

Reply:
@mebeim I'm sympathetic to that concern, but I'm afraid that ship has sailed long ago, as most translations have been receiving tweaks during the PR review process that made them deviate from a direct translation from the English version. Avoiding an out-of-sync issue in this particular page would not really ameliorate current our situation.

(That does raise an interesting challenge for the translation setup, by the way: what if someone suggests an improvement to a page that's not in English, say Korean or Chinese? Do we stop the PR and require the changes to be made in English first, and only then allow the equivalent change in the original language? That adds a bunch of friction to the contribution process, so we need to consider that carefully.)

Reply:
(That said, I don't object to this PR in particular being merged without the suggested changes, because the suggested improvements, while indeed a net positive, are not that critical — the page is quite serviceable as it is.)

Reply:
I am not saying that we should not improve translations, but this is changing the phrase and it's meaning, it's not "improving the translation". It makes the translation go out of sync with the original English version when it could be avoided. These pages seem to me already translated just fine without additional changes.

In all the Italian pages I translated and reviewed, I always made sure to drift as little as possible from the original phrases. Of course there are instances in which that does not make sense, and of course there are cases in which the English wording already is bad (I've opened a few English PRs while working on translations before opening the Italian PR because the English wording was bad).

The fact that translations already exist that have this "problem" doesn't mean we need to ignore it. If we ever manage to migrate to a sustainable translation workflow, this also makes that harder.

Reply:
Good points. I agree that we should keep translations close to the English version as long as that does not impair the translation quality, which, as you point out, was the case here.